qcodes/dataset/sqlite_base.py

```diff
@@ -485,6 +485,26 @@ def init_db(conn: SomeConnection)->None:
         transaction(conn, _layout_table_schema)
         transaction(conn, _dependencies_table_schema)
 
+
+def column_in_table(conn: SomeConnection, table: str, column: str) -> bool:
```

Comment:
Would this be better named `is_column_in_table` ?

Reply:
sure, that's better!